### PR TITLE
hero search

### PR DESCRIPTION
**Pull request by nazrulislambhat**

### CODE REVIEW

File: app/components/HeroSection/HeroSection.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    
+    const [searchQuery, setsearchQuery] = useState<string>('');
+    const handleSearchChange = (e: ChangeEvent<HTMLInputElement>) => {
+        setsearchQuery(e.target.value);
+    };
+    
+    return (
+          <div className="hero-section px-0 py-14 mx-28 custom-bg" style={{ backgroundImage: `url(${openhImage.src})` }}>
+            <div className='px-10 py-8'>
+                <h1 className='text-5xl text-white font-bold pb-2'>Welcome</h1>
```
Comment (sreenivasbttv) [inline app/components/HeroSection/HeroSection.tsx]: @nazrulislambhat To enhance component reusability, make an effort to utilize props whenever feasible. In this instance, consider converting the welcome text into a prop to gain more comprehensive control over the component.

File: app/components/HeroSection/HeroSection.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                <p className='text-2xl text-white font-semibold pb-12'>Millions of movies, TV shows and people to discover. Explore now.</p>
```
Comment (sreenivasbttv) [inline app/components/HeroSection/HeroSection.tsx]: @nazrulislambhat also convert this text to prop

**Review by sreenivasbttv — DISMISSED** [review]
Please check my comments and fix as requested

File: app/components/HeroSection/HeroSection.tsx (hunk 2 above)
Comment (nazrulislambhat) [reply app/components/HeroSection/HeroSection.tsx]: Resolved

File: app/components/HeroSection/HeroSection.tsx (hunk 1 above)
Comment (nazrulislambhat) [reply app/components/HeroSection/HeroSection.tsx]: Resolved

**Pull request merged**